zulipterminal/ui_tools/views.py

```diff
@@ -245,6 +245,19 @@ def read_message(self, index: int=-1) -> None:
         self.model.mark_message_ids_as_read(read_msg_ids)
 
 
+class StreamsViewDivider(urwid.Divider):
+    """
+    A custom urwid.Divider to visually separate pinned streams from the
+    unpinned streams.
+    """
+    def __init__(self, div_char: str='-', top: int=0, bottom: int=0) -> None:
```

Comment:
Unlike the previous implementation, I have set the `div_char='-'` from the `__init__` instead. This shall help to maintain consistency (between `StreamsView` and `LeftColumnView` stream view dividers) if we ever wish to update the `div_char`.